src/researches/findKeywordInPageTitle.js

```diff
@@ -13,12 +13,28 @@ var escapeRegExp = require( "lodash/escapeRegExp" );
  */
 
 module.exports = function( paper ) {
-	var title = paper.getTitle();
-	var keyword = escapeRegExp( paper.getKeyword() );
-	var locale = paper.getLocale();
-	var result = { matches: 0, position: -1 };
-	result.matches = wordMatch( title, keyword, locale );
-	result.position = title.toLocaleLowerCase().indexOf( keyword );
+	const title = paper.getTitle();
+	// const locale = paper.getLocale();
```

Comment:
Even for the prototype, I suggest you make sure that the morphological forms are only built when the locale is English. Currently it unnecessarily builds forms for other languages as well (according to the English pattern). I think throwing out the basic functionality for the other languages now will make it more difficult to restore it when we actually want to release the morphological analysis for English.

Reply:
Fixed.